app/models/druid_version_zip.rb

```diff
@@ -166,6 +166,10 @@ def zip_split_size
     '10g'
   end
 
+  def moab_version_size
+    moab_version_files.sum { |f| File.size(f) }
+  end
```

Comment:
Since `DruidVersionZip` already has a method to get the size on a disk of a given druid + version + storage location, I decided to use that method instead of writing a new one. And as part of that, I moved this method into the public API (and added a test below).